### PR TITLE
 frontend: convert bitbox02bootloader.tsx to a function component 

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: frontends/web/src/api/bitbox02bootloader.ts [hunk 1]
```diff
@@ -41,7 +42,7 @@ export type TVersionInfo = {
 
 export const getVersionInfo = (
   deviceID: string,
-): Promise<TStatus> => {
+): Promise<TVersionInfo> => {
```
Comment (thisconnect) [inline frontends/web/src/api/bitbox02bootloader.ts:45]: 👍  (looks like this was not used anywhere before)

File: frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx [hunk 2]
```diff
@@ -14,165 +15,109 @@
...
     );
   }
-}
-
-const loadHOC = load<LoadedProps, BitBox02BootloaderProps & TranslateProps>(({ deviceID }) => ({ versionInfo: 'devices/bitbox02-bootloader/' + deviceID + '/version-info' }))(BitBox02Bootloader);
-const HOC = translate()(loadHOC);
-export { HOC as BitBox02Bootloader };
+  return (
+    <CenteredContent>
+      {getDarkmode() ? <BitBox02Inverted /> : <BitBox02 />}
```
Comment (thisconnect) [inline frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx:116]: nit: with functional component we could now use darkmode hook.

```js
import { useDarkmode } from '../hooks/darkmode';

const { isDarkMode } = useDarkmode();
```

**Review by thisconnect — APPROVED** [review]
untested LGTM 🎉

File: frontends/web/src/api/bitbox02bootloader.ts (hunk 1 above)
Comment (benma) [reply frontends/web/src/api/bitbox02bootloader.ts:45]: it was me in the previous commit in this same PR :facepalm: fixed in the right commit :)

File: frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx (hunk 2 above)
Comment (benma) [reply frontends/web/src/components/devices/bitbox02bootloader/bitbox02bootloader.tsx]: great! done

**Pull request merged**